Author: Helge Deller <deller@gmx.de>
Description: mariadb-5.5 FTBFS, patch for hppa architecture attached #751805

--- old/storage/xtradb/os/os0stacktrace.c	2014-06-16 16:19:31.288965684 +0200
+++ new/storage/xtradb/os/os0stacktrace.c	2014-06-16 17:16:44.210474810 +0200
@@ -85,7 +85,7 @@ os_stacktrace_print(
 	caller_address = (void*) uc->uc_mcontext.gregs[REG_RIP] ;
 #elif defined(__hppa__)
 	ucontext_t* uc = (ucontext_t*) ucontext;
-	caller_address = (void*) uc->uc_mcontext.sc_iaoq[0] & ~0×3UL ;
+	caller_address = (void*) (uc->uc_mcontext.sc_iaoq[0] & ~0x3UL);
 #elif (defined (__ppc__)) || (defined (__powerpc__))
 	ucontext_t* uc = (ucontext_t*) ucontext;
 	caller_address = (void*) uc->uc_mcontext.regs->nip ;
