Author: Otto Kekäläinen <otto@seravo.fi>
Bug: https://mariadb.atlassian.net/browse/MDEV-7359
Subject: MTR tests failed to start due to replace binary not found
Last-Update: 2014-12-28

--- old/mysql-test/mysql-test-run.pl
+++ new/mysql-test/mysql-test-run.pl
@@ -2490,14 +2490,6 @@ sub environment_setup {
                                  "$bindir/sql$opt_vs_config/mysql_tzinfo_to_sql");
   $ENV{'MYSQL_TZINFO_TO_SQL'}= native_path($exe_mysql_tzinfo_to_sql);
 
-  # ----------------------------------------------------
-  # replace
-  # ----------------------------------------------------
-  my $exe_replace= mtr_exe_exists(vs_config_dirs('extra', 'replace'),
-                                 "$basedir/extra/replace",
-                                 "$path_client_bindir/replace");
-  $ENV{'REPLACE'}= native_path($exe_replace);
-
   # Create an environment variable to make it possible
   # to detect that valgrind is being used from test cases
   $ENV{'VALGRIND_TEST'}= $opt_valgrind;
