--- old/mysql-test/suite/plugins/t/unix_socket.test	2013-05-14 12:49:52 +0000
+++ new/mysql-test/suite/plugins/t/unix_socket.test	2014-02-05 10:25:37 +0000
@@ -3,10 +3,23 @@
 # get .result differences from CURRENT_USER().
 --source include/not_as_root.inc
 
+# The previous check verifies that the user does not have root permissions. 
+# However in some cases tests are run under a user named 'root',
+# even although this user does not have real root permissions. 
+# This test should be skipped in this case, since it does not expect
+# that there are records in mysql.user where user=<username>
+if ($USER=="root") {
+  skip Cannot be run by user named 'root' even if it does not have all privileges;
+}
+
 if (!$AUTH_SOCKET_SO) {
   skip No auth_socket plugin;
 }
 
+if (!$USER) {
+  skip USER variable is undefined;
+}
+
 let $plugindir=`SELECT @@global.plugin_dir`;
 
 eval install plugin unix_socket soname '$AUTH_SOCKET_SO';
