--- old/mysql-test/CMakeLists.txt	2014-02-12 17:55:23.469136898 +0200
+++ new/mysql-test/CMakeLists.txt	2014-02-12 17:54:07.471406742 +0200
@@ -64,7 +64,7 @@
 ENDIF()
 
 
+SET(MTR_FORCE perl ./mysql-test-run.pl --force --testcase-timeout=30)
-SET(MTR_FORCE perl ./mysql-test-run.pl --force)
 IF(EXISTS ${CMAKE_SOURCE_DIR}/mysql-test/suite/nist)
   SET(TEST_NIST ${MTR_FORCE} --comment=nist suite=nist ${EXP} && 
     ${MTR_FORCE} --comment=nist --force --suite=nist+ps ${EXP})
