Author: Otto Kekäläinen <otto@seravo.fi>
Description: Remove upstream debian/* handling that is obsolete

--- old/CMakeLists.txt	2014-01-29 20:55:56.499842093 +0200
+++ new/CMakeLists.txt	2014-01-31 17:12:27.633594128 +0200
@@ -430,12 +430,6 @@
 CONFIGURE_FILE(
     ${CMAKE_SOURCE_DIR}/cmake/info_macros.cmake.in ${CMAKE_BINARY_DIR}/info_macros.cmake @ONLY)
 
-IF(DEB)
-  CONFIGURE_FILE(
-    ${CMAKE_SOURCE_DIR}/debian/mariadb-server-5.5.files.in
-    ${CMAKE_SOURCE_DIR}/debian/mariadb-server-5.5.files)
-ENDIF(DEB)
-
 # Handle the "INFO_*" files.
 INCLUDE(${CMAKE_BINARY_DIR}/info_macros.cmake)
 # Source: This can be done during the cmake phase, all information is
